src/plugin/sdl/sdl.c

```diff
@@ -279,11 +328,7 @@ int SDL_init(void)
   }
 #endif
 
-#ifdef X_SUPPORT
-  init_x11_support(window);
-#else
-  use_bitmap_font = 1;
-#endif
+  SDL_SetWindowMinimumSize(window, MIN_X, MIN_Y);
```

Comment:
I think its better to do in `SDL_change_mode()`.
It already does `SDL_SetWindowSize(window, w_x_res, w_y_res);`
(3 times), and you can add  `SDL_SetWindowMinimumSize(window, x_res, y_res);`.